src/redux/auth/reducers.js

```diff
@@ -82,7 +82,9 @@ const auth = (state = initialState, action = {}) => {
         errorMessage: errorMessageHandling(payload),
       };
     // LOGOUT_USER
-    case constants.LOGOUT_USER:
+    case constants.LOGOUT_USER_FULFILLED:
```

Comment:
I don't know but I maybe even on error it is best to logout?

Reply:
Then the user wouldn't be logged out in the backend... I'm not sure if this is best practice.